### PR TITLE
Restapi

### PR DESCRIPTION
**Pull request by Wlh0**

### CODE REVIEW

File: configuration/conf/config.properties [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 # 主机列表，如果有多个主机则使用英文逗号进行分割
 # HOST=127.0.0.1
-
+# REST_AUTHORIZATION=Basic cm9vdDpyb290
```
Comment (SpriCoder) [inline configuration/conf/config.properties:11]: Please add a comment of REST_AUTHORIZATION

File: iotdb-1.3/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb130/IoTDBRestAPI.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import cn.edu.tsinghua.iot.benchmark.entity.Batch.IBatch;
+import cn.edu.tsinghua.iot.benchmark.entity.Record;
+import cn.edu.tsinghua.iot.benchmark.entity.Sensor;
+import cn.edu.tsinghua.iot.benchmark.exception.DBConnectException;
+import cn.edu.tsinghua.iot.benchmark.measurement.Status;
+import cn.edu.tsinghua.iot.benchmark.schema.schemaImpl.DeviceSchema;
+import cn.edu.tsinghua.iot.benchmark.tsdb.DBConfig;
+import cn.edu.tsinghua.iot.benchmark.tsdb.TsdbException;
+import com.google.gson.Gson;
+import okhttp3.*;
```
Comment (SpriCoder) [inline iotdb-1.3/src/main/java/cn/edu/tsinghua/iot/benchmark/iotdb130/IoTDBRestAPI.java:15]: Please not use * here

**Review by SpriCoder — CHANGES_REQUESTED** [review]
Some minor problems, PTAL~

**Review by SpriCoder — APPROVED** [review]
LGTM, And please paste a test result

**Comment by Wlh0** [comment]
img_v3_02c5_1b0a6564-f249-4f75-bcc7-4ddab283f4eg
img_v3_02c5_d15266b4-0fde-48da-aaca-2ee445a2d74g

**Pull request merged**